helm-chart/kubemod/crds/crd-kubemod.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
We should reflect this on the minimum version of kubernetes in chart.yaml.

Reply:
Good idea... will make the change

Reply:
I have pushed this. Sorry for the delay.

Reply:
Thank you @llamashoes!

I think there's only one thing left - @desaintmartin's suggestion to add `kubeVersion` field to Chart.yaml and set it to greater than or equal to Kubernetes 1.16, where v1beta1 CRDs were deprecated.

Please see https://helm.sh/docs/topics/charts/#the-kubeversion-field

Reply:
Sweet! Looking now...

Reply:
Thanks @llamashoes !